docs/architecture/page.rst

```diff
@@ -0,0 +1,41 @@
+Saleor Pages
+===================
+
+
+Setting up custom pages
+-----------------------
+
+You can set up your own pages such as "About us" or "Important Announcement!" in the Pages menu in dashboard.
+Note that if you are not an admin, you need to be in group with proper permissions.
+
+
+Providing an URL to your page in storefront
+-------------------------------------------
+
+If you want to add a link to recently created page in storefront, all you need to do is to put the following code:
+
+.. code-block:: html
+
+  <a href="{% url "page:details" slug="your-slug" %}">Text for the link</a>
+
+in the proper template.
+For example in default Saleor setup, in the footer there is a list of categories:
```

Comment:
The example above already explains how to add a link to a static page in the storefront. This whole example seems redundant. I think we could even improve the example above and instead of "your-slug" and "Text for the link" use real-life values like "terms-of-service" and "Terms of Service".